docs/cugraph/source/installation/source_build.md

```diff
@@ -1,53 +1,46 @@
 # Building from Source
 
-The following instructions are for users wishing to build cuGraph from source code.  These instructions are tested on supported distributions of Linux, CUDA, and Python - See [RAPIDS Getting Started](https://rapids.ai/start.html) for list of supported environments.  Other operating systems _might be_ compatible, but are not currently tested.
-
-The cuGraph package include both a C/C++ CUDA portion and a python portion.  Both libraries need to be installed in order for cuGraph to operate correctly.
+These instructions are tested on supported distributions of Linux, CUDA, and
+Python - See [RAPIDS Getting Started](https://rapids.ai/start.html) for list of
+supported environments.  Other operating systems _might be_ compatible, but are
+not currently tested.
 
 ## Prerequisites
 
 __Compiler:__
 * `gcc`           version 9.3+
 * `nvcc`          version 11.0+
-* `cmake`         version 3.20.1+
 
 __CUDA:__
 * CUDA 11.0+
 * NVIDIA driver 450.80.02+
 * Pascal architecture or better
 
-You can obtain CUDA from [https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
-
-__Packages:__
-* `cmake`         version 3.20.1+
-* `libcugraphops` (version matching source branch version, eg. `23.10`)
-
-You can obtain `libcugraphops` using `conda`/`mamba` from the `nvidia` channel, or using `pip` with the `--extra-index-url=https://pypi.nvidia.com` option.  See the [RAPIDS docs](https://docs.rapids.ai/install#environment) for more details.
-
-## Building cuGraph
-To install cuGraph from source, ensure the dependencies are met.
+You can obtain CUDA from
+[https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
```

Comment:
This points to cuda 12.3.
should it point to the all versions page or maybe 12.0  ? Are we supporting 12.3 ?
Rapids download page only supports up to 12.0

Reply:
Thanks. It looks like that link was copied from [cuDF's version of this page](https://github.com/rapidsai/cudf/blob/branch-24.02/CONTRIBUTING.md#setting-up-your-build-environment) (cc @GregoryKimball ).  Instead of trying to keep this page up-to-date, I just linked to the RAPIDS sys requirements page which has the proper download links plus much more relevant info.